sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/SensorContextUtils.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * SonarTS
+ * Copyright (C) 2017-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugin.typescript;
+
+import org.sonar.api.batch.fs.FilePredicate;
+import org.sonar.api.batch.fs.FileSystem;
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.batch.sensor.SensorContext;
+import org.sonar.api.batch.sensor.issue.NewIssue;
+import org.sonar.api.batch.sensor.issue.NewIssueLocation;
+import org.sonar.api.rule.RuleKey;
+
+public class SensorContextUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1118))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=sonarlint-integration-poc&issues=AWDbX2TkFjpmjnDt3Bpe&open=AWDbX2TkFjpmjnDt3Bpe)

[ISSUE_KEY]: AWDbX2TkFjpmjnDt3Bpe